NEWS.md

```diff
@@ -1,251 +0,0 @@
-## DataFrames v0.6.11 Release Notes
```

Comment:
A for effort. Sorry, I didn't actually mean to suggest updating the release notes of the old releases (although I loved the idea). Just wanted to clarify that we could distribute all release info going forward via the notes on the Release page (never made a package release so I'm completely unfamiliar with the process e.g. is there a low character limit or something prohibitively restrictive). Unless someone more knowledgeable says otherwise, I think just removing this file and distributing release info with the releases is a great strategy going forward. Carry on with the great PR